library/bignum_core.c

```diff
@@ -525,4 +525,32 @@ int mbedtls_mpi_core_get_mont_r2_unsafe( mbedtls_mpi *X,
     return( ret );
 }
 
+
+/* Fill X with n_bytes random bytes.
+ * X must already have room for those bytes.
+ * The ordering of the bytes returned from the RNG is suitable for
+ * deterministic ECDSA (see RFC 6979 §3.3 and the specification of
+ * mbedtls_mpi_core_random()).
```

Comment:
`mbedtls_mpi_core_random` doesn't exist yet, it will be added by the follow-up https://github.com/Mbed-TLS/mbedtls/pull/6303.